packages/sync-repo-settings/src/bin/sync-repo-settings.ts

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import yargs = require('yargs');
+import {remoteSync} from './commands/remote-sync';
+import {localSync} from './commands/local-sync';
+
+yargs(process.argv.slice(2))
```

Comment:
I don't think two distinct commands are strictly necessary here.  Why not one command?  If they pass `--file`, we use the local approach, otherwise assume remote?

Reply:
Ok, can do.